format everything of the applications

matthias11-21 opened the pull request:
Format everything of the applications instead of just formatting the source directory.  This will include formatting of `include` and `test` directories.

matthias11-21 committed:
format everything of the applications

tsloughter commented:
Won't this end up going into `_build` too? It should use the `src_dirs` and `extra_src_dirs` lists of configured directories plus the `test` directory.

matthias11-21 committed:
fix: prevent formatting of files in _build

If working with an application (instead of a release) the
modification would format files in _build, too.  Prevent
that by formatting the directories src, test, and include
explicitly.

Using rebar3_dir:all_src_dirs/1 is way more trickier as
this function reads the values from the rebar configuration
which does not include sane defaults or is dependent on
profiles.  We just want to format everything.

matthias11-21 commented:
After looking at what it needs to use `rebar_dir` and `src_dirs` I went the much simpler route mimicking the original approach.  Just format `src`, `include`, and `test`.  I don't want to format depending on target anyway.  I want to format every Erlang source file.
